File: Programmers/Level1/3진법 뒤집기.py
def solution(n):
    answer = 0
    remainder = []
    while(n >= 3):
        remainder.append(n % 3)
        n //= 3
    remainder.append(n)
    remainder.reverse()

    for i in range(len(remainder)):
        answer += remainder[i] * (3 ** i)

    return answer

File: Swift 코딩테스트 연습/3진법 뒤집기.swift
import Foundation

func solution(_ n: Int) -> Int {
    var numThree = Array(String(n, radix: 3)).map{Int(String($0))!}
    var answer: Int = numThree[0]

    for i in 1.. < numThree.count {
        var value = pow(3.0, i)
        var nsValue = NSDecimalNumber(decimal: value)
        answer += numThree[i] * Int(nsValue)
    }

    return answer
}
